### PR TITLE
fix: code display -> table display

### PR DESCRIPTION
**Pull request by imsunhao**
i find this bug and try to fix it

### CODE REVIEW

**Comment by tengqm** [comment]
@imsunhao pls sign the CLA in order for this to be reviewed.

**Review by Arhell — COMMENTED** [review]
@imsunhao 
1. Login to the [CLA site](https://identity.linuxfoundation.org/projects/cncf) and change it to match what you use as your default email address on your GitHub account
2. Change your default email address on your GH account to the same one you used to sign the CLA.

**Comment by tengqm** [comment]
What happened? What bug this PR is fixing?

File: content/zh/docs/reference/kubectl/kubectl.md [hunk 1]
```diff
@@ -32,441 +32,443 @@ kubectl [flags]
...
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">
-      <!--
-      comma-separated list of pattern=N settings for file-filtered logging
-      -->
-      以逗号分隔的 pattern=N 设置列表，用于过滤文件的日志记录
-      </td>
-    </tr>
+
+  <table style="width: 100%; table-layout: fixed;">
```
Comment (irvifa) [inline content/zh/docs/reference/kubectl/kubectl.md:36]: Probably an explanation on what we're trying to fix will be helpful. At least put the explanation in commit message or description on the PR.

**Comment by tengqm** [comment]
closing since no response for over 1 month.
/close

**Pull request closed without merging**